reconcile/aws_cloudwatch_log_retention/integration.py

```diff
@@ -230,17 +239,40 @@ def _reconcile_log_groups(
             )
 
 
-def get_active_aws_accounts() -> list[dict]:
-    return [
-        a
-        for a in get_aws_accounts(cleanup=True)
-        if "aws-cloudwatch-log-retention"
-        not in (a.get("disable") or {}).get("integrations", [])
-    ]
+# def get_aws_accounts_test() -> list[dict]:
+#     gql_api_query = gql.get_api().query
+#     return [
+#         account.dict(by_alias=True)
+#         for account in get_aws_accounts(query_func=gql_api_query).accounts
+#         or []
+#     ]
+
+
+def account_to_dict(account: AWSAccountV1) -> dict[str, Any]:
+    return account.dict(by_alias=True)
+
+
+def get_active_aws_accounts() -> list[AWSAccountV1]:
+    accounts = get_aws_accounts(gql.get_api())
+    if accounts is not None:
+        accounts = [
+            account
+            for account in accounts
+            if account.disable is None
+            or (
+                account.disable.integrations is not None
+                and (
+                    "aws-cloudwatch-log-retention" not in (account.disable.integrations)
+                )
+            )
```

Comment:
none check can be simplified like this

```python
if not (account.disable and account.disable.integrations and "aws-cloudwatch-log-retention" in account.disable.integrations)
```

and inline return directly instead of overwrite `accounts`, so 

```python
        return [
            account
            for account in accounts
            if not (account.disable and account.disable.integrations and "aws-cloudwatch-log-retention" in account.disable.integrations)
         ]
```